### ISSUE TITLE
BUG: 

### ISSUE THREAD

**Issue by gwimmer** [firedrakeproject/firedrake#3107]
**Describe the bug**
`HybridizedPC` does not work with periodic extruded meshes

**Steps to Reproduce**
Call 'firedrake.HybridizationPC' in solver_parameters for any problem with a periodic extruded mesh

**Error message**
Firedrake complains about the top an bottom boundaries

**Additional Info**
This might be fixable by adding an exception including `mesh.cell_set._extruded_periodic` when adding the `top` and `bottom` markers in the `initialize` method of the `HybridizedPC` class

**Comment by ksagiyam** [comment]
Yes. There is no `top` or `bottom` boundaries for the periodic case, so all horizontal facets simply need to be treated the same way, I think. (We left `bottom` boundary available, but that is only for applying boundary conditions).
